src/imp/tracer_imp.js

```diff
@@ -183,24 +184,57 @@ export default class TracerImp extends EventEmitter {
     }
 
     startSpan(fields) {
-        let spanImp = new SpanImp(this);
+        // First, assemble the SpanContextImp for our SpanImp.
+        //
+        // Normalize `fields.reference` and `fields.references` (at most one
+        // will be defined).
+        let refs = fields.references;
+        if (fields.reference) {
+            refs = [fields.reference];
+        }
```

Comment:
I know there's an open issue on whether there should be separate interface/implementation objects in the OpenTracing API, but one of the intentions of the interface layer was to handle argument normalization *at that layer* rather than have implementations do it. In other words, `reference` should be normalized to a single element `references` array before this call is made.